go-sdk/worker/runner.go

```diff
@@ -342,6 +342,7 @@ func (w *worker) ExecuteTaskWorkload(ctx context.Context, workload api.ExecuteTa
 			EndDate: endTime,
 			State:   api.TISuccessStatePayloadState(finalState),
 		})
+
```

Comment:
I'm guessing this was to have a change to trigger the pre-commit stuff, right? Once we are happy with it can we undo this otherwise pointless change?